ftdc/custom_format.go

```diff
@@ -118,7 +118,64 @@ func writeDatum(time int64, prev, curr []float32, output io.Writer) {
 	}
 }
 
-// getFieldsForItem returns the (flattened) list of strings for a metric structure. For example the
+var errNotStruct = errors.New("stats object is not a struct")
+
+func isNumeric(kind reflect.Kind) bool {
```

Comment:
Odd that this doesn't exist in the `reflect` package.

Reply:
I assume it's because there's no "number" interface. Meaning, even if they had a `reflect.IsNumeric` method, what good would it do? There's no `reflect.GetAsNumeric`.

That said, I think they could be a bit more ergonomic about this.